### PR TITLE
CppCheck StandardRatings and various through Humidifiers

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes programming errors
 - Use CppCheck as guide

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/StandardRatings.cc [hunk 1]
```diff
@@ -4508,7 +4508,7 @@ namespace StandardRatings {
                                 p_int, q_int, q_low, bl, n, Q_E_Int(spnum), q_full, P_E_Int(spnum), p_full, p_low);
                             speedsUsed.push_back(spnum);
                             goto SpeedLoop3_exit;
-                        } else if (bl < q_full) {
+                        } else { // bl < q_full
```
Comment (rraustad) [inline src/EnergyPlus/StandardRatings.cc:4498]: [src/EnergyPlus/StandardRatings.cc:4511]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.

File: src/EnergyPlus/StandardRatings.cc [hunk 2]
```diff
@@ -6469,7 +6469,7 @@ namespace StandardRatings {
                 } else if (bl >= q_full) {
                     // CASE 3 : 11.2.2.3.3 AHRI-2023
                     // Building Load is greater than the capacity of the unit at the Full Compressor Speed, q_full <= bl or (bl >= q_full:)
-                    if (t > (-15) || t <= (-8.33)) {
+                    if (t > (-15) || t <= (-8.33)) { // Logical disjunction always evaluates to true: t > -15 || t <= -8.33
```
Comment (rraustad) [inline src/EnergyPlus/StandardRatings.cc:6447]: Didn't want to change the answer

[src/EnergyPlus/StandardRatings.cc:6472]:(warning),[incorrectLogicOperator],Logical disjunction always evaluates to true: t > -15 || t <= -8.33.

File: src/EnergyPlus/StandardRatings.cc [hunk 3]
```diff
@@ -4606,7 +4606,7 @@ namespace StandardRatings {
                             std::tie(q, e, NetTotCoolCapBinned_2023, TotCoolElecPowerBinned_2023) = IntermediateCapacityAndPowerSEER2Case2A(
                                 p_int, q_int, q_low, bl, n, Q_E_Int(spnum), q_full, P_E_Int(spnum), p_full, p_low);
                             goto SpeedLoop3_exit;
-                        } else if (bl < q_full) {
+                        } else { // bl < q_full
```
Comment (rraustad) [inline src/EnergyPlus/StandardRatings.cc:4596]: [src/EnergyPlus/StandardRatings.cc:4609]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.
[

File: src/EnergyPlus/StandardRatings.cc [hunk 4]
```diff
@@ -4647,7 +4647,7 @@ namespace StandardRatings {
...
```
Comment (rraustad) [inline src/EnergyPlus/StandardRatings.cc:4637]: [src/EnergyPlus/StandardRatings.cc:4650]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.

File: src/EnergyPlus/StandardRatings.cc [hunk 5]
```diff
@@ -4691,7 +4691,7 @@ namespace StandardRatings {
                         std::tie(q, e, NetTotCoolCapBinned_2023, TotCoolElecPowerBinned_2023) =
                             IntermediateCapacityAndPowerSEER2Case2A(p_int, q_int, q_low, bl, n, Q_E_Int(1), q_full, P_E_Int(1), p_full, p_low);
                         goto SpeedLoop3_exit;
-                    } else if (bl < q_full) {
+                    } else { // bl < q_full
```
Comment (rraustad) [inline src/EnergyPlus/StandardRatings.cc:4681]: [src/EnergyPlus/StandardRatings.cc:4694]:(warning),[identicalInnerCondition],Identical inner 'if' condition is always true.

File: src/EnergyPlus/Humidifiers.cc [hunk 6]
```diff
@@ -742,31 +742,27 @@ namespace Humidifiers {
...
+        Real64 AirDensity;       // Density of air
 
         if (HumType == HumidType::Electric || HumType == HumidType::Gas) {
-            IsAutoSize = false;
-            HardSizeNoDesRun = false;
+            bool IsAutoSize = false;
+            bool HardSizeNoDesRun = false;
             NomPowerDes = 0.0;
             NomPowerUser = 0.0;
+            std::string ModuleObjectType; // for ease in getting objects
```
Comment (rraustad) [inline src/EnergyPlus/Humidifiers.cc:753]: [src/EnergyPlus/Humidifiers.cc:745]:(style),[variableScope],The scope of the variable 'ModuleObjectType' can be reduced.
[src/EnergyPlus/Humidifiers.cc:746]:(style),[variableScope],The scope of the variable 'RefrigerantIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:747]:(style),[variableScope],The scope of the variable 'WaterIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:752]:(style),[variableScope],The scope of the variable 'IsAutoSize' can be reduced.
[src/EnergyPlus/Humidifiers.cc:753]:(style),[variableScope],The scope of the variable 'HardSizeNoDesRun' can be reduced.

File: src/EnergyPlus/Humidifiers.cc [hunk 7]
```diff
@@ -1189,8 +1185,6 @@ namespace Humidifiers {
         Real64 SteamSatEnthalpy;        // enthalpy of saturated steam at 100C [J/kg]
         Real64 WaterSatEnthalpy;        // enthalpy of saturated water at 100C [J/kg]
         Real64 Tref;                    // humidifier entering water temperature [C]
-        int RefrigerantIndex;           // refiferant index
-        int WaterIndex;                 // fluid type index
```
Comment (rraustad) [inline src/EnergyPlus/Humidifiers.cc:1193]: [src/EnergyPlus/Humidifiers.cc:1192]:(style),[variableScope],The scope of the variable 'RefrigerantIndex' can be reduced.
[src/EnergyPlus/Humidifiers.cc:1193]:(style),[variableScope],The scope of the variable 'WaterIndex' can be reduced.

File: src/EnergyPlus/Humidifiers.cc [hunk 8]
```diff
@@ -150,6 +149,7 @@ namespace Humidifiers {
                     state,
                     format("SimHumidifier: Invalid CompIndex passed={}, Number of Units={}, Entered Unit name={}", HumNum, NumHumidifiers, CompName));
             }
+            auto &CheckEquipName = state.dataHumidifiers->CheckEquipName;
```
Comment (rraustad) [inline src/EnergyPlus/Humidifiers.cc]: [src/EnergyPlus/Humidifiers.cc:132]:(style),[variableScope],The scope of the variable 'CheckEquipName' can be reduced.

Looks like I missed this one. Next pass.

[src/EnergyPlus/Humidifiers.cc:132]:(style),[constVariable],Variable 'CheckEquipName' can be declared as reference to const

File: src/EnergyPlus/HighTempRadiantSystem.cc [hunk 9]
```diff
@@ -950,16 +950,16 @@ namespace HighTempRadiantSystem {
...
-                        ZoneTemp = thisZoneHB.MAT;
+                        ZoneTemp = thisZoneHBMod.MAT;
                     } break;
                     case RadControlType::MRTControl: {
-                        ZoneTemp = thisZoneHB.MRT;
+                        ZoneTemp = thisZoneHBMod.MRT;
                     } break;
                     case RadControlType::OperativeControl: {
-                        ZoneTemp = 0.5 * (thisZoneHB.MAT + thisZoneHB.MRT);
+                        ZoneTemp = 0.5 * (thisZoneHBMod.MAT + thisZoneHBMod.MRT);
```
Comment (rraustad) [inline src/EnergyPlus/HighTempRadiantSystem.cc:962]: [src/EnergyPlus/HighTempRadiantSystem.cc:953]:(style),[shadowVariable],Local variable 'thisZoneHB' shadows outer variable
[src/EnergyPlus/HighTempRadiantSystem.cc:814]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const
[src/EnergyPlus/HighTempRadiantSystem.cc:907]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const
[src/EnergyPlus/HighTempRadiantSystem.cc:953]:(style),[constVariable],Variable 'thisZoneHB' can be declared as reference to const

File: src/EnergyPlus/HeatBalanceSurfaceManager.cc [hunk 10]
```diff
@@ -9056,7 +9056,7 @@ void CalcHeatBalanceInsideSurf2CTFOnly(EnergyPlusData &state,
 void sumSurfQdotRadHVAC(EnergyPlusData &state)
 {
     for (int surfNum : state.dataSurface->allGetsRadiantHeatSurfaceList) {
-        auto &thisSurfQRadFromHVAC = state.dataHeatBalFanSys->surfQRadFromHVAC(surfNum);
+        auto const &thisSurfQRadFromHVAC = state.dataHeatBalFanSys->surfQRadFromHVAC(surfNum);
         state.dataHeatBalSurf->SurfQdotRadHVACInPerArea(surfNum) = thisSurfQRadFromHVAC.HTRadSys + thisSurfQRadFromHVAC.HWBaseboard +
                                                                    thisSurfQRadFromHVAC.SteamBaseboard + thisSurfQRadFromHVAC.ElecBaseboard +
                                                                    thisSurfQRadFromHVAC.CoolingPanel;
```
Comment (rraustad) [inline src/EnergyPlus/HeatBalanceSurfaceManager.cc:9062]: [src/EnergyPlus/HeatBalanceSurfaceManager.cc:8470]:(style),[constVariable],Variable 'Surface' can be declared as reference to const
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:8655]:(style),[constVariable],Variable 'thisSpace' can be declared as reference to const
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:9059]:(style),[constVariable],Variable 'thisSurfQRadFromHVAC' can be declared as reference to const

File: src/EnergyPlus/Fans.cc [hunk 11]
```diff
@@ -1343,8 +1343,7 @@ void FanComponent::set_size(EnergyPlusData &state)
     static constexpr std::string_view routineName = "FanComponent::set_size()"; // include trailing blank space
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    bool _bPRINT = true;  // TRUE if sizing is reported to output (eio)
-    int NumFansSized = 0; // counter used to deallocate temporary string array after all fans have been sized
+    bool _bPRINT = true; // TRUE if sizing is reported to output (eio)
```
Comment (rraustad) [inline src/EnergyPlus/Fans.cc:1346]: [src/EnergyPlus/Fans.cc:849]:(style),[shadowVariable],Local variable 'cCurrentModuleObject' shadows outer variable
[src/EnergyPlus/Fans.cc:1347]:(style),[unreadVariable],Variable 'NumFansSized' is assigned a value that is never used.

File: src/EnergyPlus/EvaporativeCoolers.cc [hunk 12]
```diff
@@ -3490,12 +3488,6 @@ void GetInputZoneEvaporativeCoolerUnit(EnergyPlusData &state)
             // with used for ZONECOOLINGLOADVARIABLESPEEDFAN Cooler Unit Control Method
             thisZoneEvapUnit.fanOp = HVAC::FanOp::Cycling;
 
-            FanVolFlow = 0.0;
-            if (errFlag) {
-                ShowContinueError(state, format("specified in {} = {}", CurrentModuleObject, thisZoneEvapUnit.Name));
-                ErrorsFound = true;
-            }
-
```
Comment (rraustad) [inline src/EnergyPlus/EvaporativeCoolers.cc:3498]: [src/EnergyPlus/EvaporativeCoolers.cc:3494]:(style),[knownConditionTrueFalse],Condition 'errFlag' is always false
[src/EnergyPlus/EvaporativeCoolers.cc:1187]:(style),[variableScope],The scope of the variable 'SizingDesRunThisZone' can be reduced.

File: src/EnergyPlus/EconomicTariff.cc [hunk 13]
```diff
@@ -3879,9 +3879,6 @@ void LEEDtariffReporting(EnergyPlusData &state)
     Real64 distHeatWaterTotalCost;
     Real64 distHeatSteamTotalCost;
     Real64 allTotalCost;
-    std::string distCoolTariffNames;
-    std::string distHeatWaterTariffNames;
-    std::string distHeatSteamTariffNames;
     EconConv elecUnits;
     EconConv gasUnits;
     EconConv distCoolUnits;
```
Comment (rraustad) [inline src/EnergyPlus/EconomicTariff.cc:3884]: [src/EnergyPlus/EconomicTariff.cc:3926]:(style),[shadowVariable],Local variable 'distCoolTariffNames' shadows outer variable
[src/EnergyPlus/EconomicTariff.cc:3927]:(style),[shadowVariable],Local variable 'distHeatWaterTariffNames' shadows outer variable
[src/EnergyPlus/EconomicTariff.cc:3928]:(style),[shadowVariable],Local variable 'distHeatSteamTariffNames' shadows outer variable
[src/EnergyPlus/EconomicTariff.cc:3882]:(style),[unusedVariable],Unused variable: distCoolTariffNames
[src/EnergyPlus/EconomicTariff.cc:3883]:(style),[unusedVariable],Unused variable: distHeatWaterTariffNames
[src/EnergyPlus/EconomicTariff.cc:3884]:(style),[unusedVariable],Unused variable: distHeatSteamTariffNames

File: src/EnergyPlus/DisplacementVentMgr.cc [hunk 14]
```diff
@@ -893,7 +892,7 @@ namespace RoomAir {
             Real64 AirCap = thisZoneHB.AirPowerCap;
             TempHistTerm = AirCap * (3.0 * thisZoneHB.ZTM[0] - (3.0 / 2.0) * thisZoneHB.ZTM[1] + OneThird * thisZoneHB.ZTM[2]);
 
-            for (Ctd = 1; Ctd <= 3; ++Ctd) {
+            for (int Ctd = 1; Ctd <= 3; ++Ctd) {
                 TempDepCoef = state.dataDispVentMgr->HA_MX + state.dataDispVentMgr->HA_OC + state.dataDispVentMgr->HA_FLOOR + MCp_Total;
```
Comment (rraustad) [inline src/EnergyPlus/DisplacementVentMgr.cc:896]: [src/EnergyPlus/DisplacementVentMgr.cc:593]:(style),[shadowVariable],Local variable 'Ctd' shadows outer variable

File: src/EnergyPlus/DesiccantDehumidifiers.cc [hunk 15]
```diff
@@ -808,7 +808,6 @@ namespace DesiccantDehumidifiers {
             ErrorsFound2 = false;
             desicDehum.RegenFanIndex = Fans::GetFanIndex(state, desicDehum.RegenFanName);
             if (desicDehum.RegenFanIndex == 0) {
-                ErrorObjectHeader eoh{routineName, CurrentModuleObject, desicDehum.Name};
```
Comment (rraustad) [inline src/EnergyPlus/DesiccantDehumidifiers.cc:811]: [src/EnergyPlus/DesiccantDehumidifiers.cc:811]:(style),[shadowVariable],Local variable 'eoh' shadows outer variable

File: src/EnergyPlus/DataZoneEquipment.cc [hunk 16]
```diff
@@ -751,10 +751,8 @@ void processZoneEquipmentInput(EnergyPlusData &state,
                                Array1D_int &NodeNums)
 {
     static constexpr std::string_view RoutineName("processZoneEquipmentInput: "); // include trailing blank space
-    std::string_view zsString = "Zone";
     int spaceFieldShift = 0;
     if (isSpace) {
-        zsString = "Space";
         spaceFieldShift = -1;
     }
```
Comment (rraustad) [inline src/EnergyPlus/DataZoneEquipment.cc:757]: [src/EnergyPlus/DataZoneEquipment.cc:754]:(style),[variableScope],The scope of the variable 'zsString' can be reduced.
[src/EnergyPlus/DataZoneEquipment.cc:754]:(style),[unreadVariable],Variable 'zsString' is assigned a value that is never used.
[src/EnergyPlus/DataZoneEquipment.cc:757]:(style),[unreadVariable],Variable 'zsString' is assigned a value that is never used.

File: src/EnergyPlus/DataZoneEquipment.cc [hunk 17]
```diff
@@ -1480,7 +1478,7 @@ void processZoneReturnMixerInput(EnergyPlusData &state,
                                       NodeInputManager::CompFluidStream::Primary,
                                       objectIsParent);
                 // Check space return nodes
-                bool found = false;
+                found = false;
```
Comment (rraustad) [inline src/EnergyPlus/DataZoneEquipment.cc:1481]: [src/EnergyPlus/DataZoneEquipment.cc:1391]:(style),[shadowVariable],Local variable 'found' shadows outer variable
[src/EnergyPlus/DataZoneEquipment.cc:1483]:(style),[shadowVariable],Local variable 'found' shadows outer variable

File: src/EnergyPlus/DataZoneEquipment.cc [hunk 18]
```diff
@@ -2274,7 +2272,7 @@ void EquipConfiguration::calcReturnFlows(EnergyPlusData &state,
                 // Establish corresponding airloop inlet(s) mass flow rate and set return node max/min/maxavail
                 Real64 inletMassFlow = 0.0;
                 int maxMinNodeNum = 0;
-                auto &thisAirLoopFlow(state.dataAirLoop->AirLoopFlow(airLoop));
+                auto const &thisAirLoopFlow(state.dataAirLoop->AirLoopFlow(airLoop));
                 if (ADUNum > 0) {
                     // Zone return node could carry supply flow to zone without leaks plus any induced flow from plenum (but don't include other
```
Comment (rraustad) [inline src/EnergyPlus/DataZoneEquipment.cc:2277]: [src/EnergyPlus/DataZoneEquipment.cc:1833]:(style),[constVariable],Variable 'zoneNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1844]:(style),[constParameter],Parameter 's' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1947]:(style),[constVariable],Variable 'spaceOutletNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:1982]:(style),[constVariable],Variable 'spaceNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2121]:(style),[constVariable],Variable 'thisZoneSysEnergyDemand' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2132]:(style),[constVariable],Variable 'equipOutletNode' can be declared as reference to const
[src/EnergyPlus/DataZoneEquipment.cc:2277]:(style),[constVariable],Variable 'thisAirLoopFlow' can be declared as reference to const

File: src/EnergyPlus/DataSurfaces.cc [hunk 19]
```diff
@@ -238,10 +238,10 @@ Real64 SurfaceData::getInsideAirTemperature(EnergyPlusData &state, const int t_S
         // determine supply air conditions
         Real64 SumSysMCp = 0;
         Real64 SumSysMCpT = 0;
-        auto &inletNodes = (state.dataHeatBal->doSpaceHeatBalance) ? state.dataZoneEquip->spaceEquipConfig(this->spaceNum).InletNode
+        auto const &inletNodes = (state.dataHeatBal->doSpaceHeatBalance) ? state.dataZoneEquip->spaceEquipConfig(this->spaceNum).InletNode
                                                                    : state.dataZoneEquip->ZoneEquipConfig(Zone).InletNode;
         for (int nodeNum : inletNodes) {
-            auto &inNode = state.dataLoopNodes->Node(nodeNum);
+            auto const &inNode = state.dataLoopNodes->Node(nodeNum);
             Real64 CpAir = PsyCpAirFnW(thisSpaceHB.airHumRat);
```
Comment (rraustad) [inline src/EnergyPlus/DataSurfaces.cc:245]: [src/EnergyPlus/DataSurfaces.cc:241]:(style),[constVariable],Variable 'inletNodes' can be declared as reference to const
[src/EnergyPlus/DataSurfaces.cc:244]:(style),[constVariable],Variable 'inNode' can be declared as reference to const

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 20]
```diff
@@ -4167,8 +4167,8 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
         auto const &zoneNode = state.dataLoopNodes->Node(zone.SystemZoneNodeNumber);
         // Calculate Grashof, Reynolds, and Richardson numbers for the zone
         // Grashof for zone air based on largest delta T between surfaces and zone height
-        for (int spaceNum : zone.spaceIndexes) {
-            auto const &thisSpace = state.dataHeatBal->space(spaceNum);
+        for (int spaceNumLoop : zone.spaceIndexes) {
+            auto const &thisSpace = state.dataHeatBal->space(spaceNumLoop);
             for (int surfNum = thisSpace.HTSurfaceFirst; surfNum <= thisSpace.HTSurfaceLast; ++surfNum) {
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:4172]: [src/EnergyPlus/ConvectionCoefficients.cc:4059]:(style),[shadowVariable],Local variable 'spaceNum' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4065]:(style),[shadowVariable],Local variable 'surface' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4068]:(style),[shadowVariable],Local variable 'DeltaTemp' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4083]:(style),[shadowVariable],Local variable 'spaceNum' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4088]:(style),[shadowVariable],Local variable 'surface' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4091]:(style),[shadowVariable],Local variable 'DeltaTemp' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4106]:(style),[shadowVariable],Local variable 'spaceNum' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4111]:(style),[shadowVariable],Local variable 'surface' shadows outer variable
[src/EnergyPlus/ConvectionCoefficients.cc:4170]:(style),[shadowVariable],Local variable 'spaceNum' shadows outer variable

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 21]
```diff
@@ -3985,7 +3985,7 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
     } else { // is controlled, lets see by how and if that means is currently active
 
         auto &zoneEquipConfig = state.dataZoneEquip->ZoneEquipConfig(surface.Zone);
-        auto &zoneNode = state.dataLoopNodes->Node(zone.SystemZoneNodeNumber);
+        auto const &zoneNode = state.dataLoopNodes->Node(zone.SystemZoneNodeNumber);
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:3988]: [src/EnergyPlus/ConvectionCoefficients.cc:3052]:(style),[constVariable],Variable 'zone' can be declared as reference to const
[src/EnergyPlus/ConvectionCoefficients.cc:3988]:(style),[constVariable],Variable 'zoneNode' can be declared as reference to const

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 22]
```diff
@@ -4056,18 +4056,18 @@ void DynamicIntConvSurfaceClassification(EnergyPlusData &state, int const SurfNu
...
+                        for (int spaceNumLoop : zone.spaceIndexes) {
+                            auto const &thisSpace = state.dataHeatBal->space(spaceNumLoop);
 
                             for (int SurfLoop = thisSpace.HTSurfaceFirst; SurfLoop <= thisSpace.HTSurfaceLast; ++SurfLoop) {
 
                                 if (!state.dataSurface->surfIntConv(SurfLoop).hasActiveInIt) continue;
-                                auto &surface = state.dataSurface->Surface(SurfLoop);
-                                if (surface.Class != SurfaceClass::Floor) continue;
+                                auto const &surfaceLoop = state.dataSurface->Surface(SurfLoop);
+                                if (surfaceLoop.Class != SurfaceClass::Floor) continue;
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:4066]: [src/EnergyPlus/ConvectionCoefficients.cc:4065]:(style),[constVariable],Variable 'surface' can be declared as reference to const

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 23]
```diff
@@ -6544,7 +6544,7 @@ Real64 SurroundingSurfacesRadCoeffAverage(EnergyPlusData &state, int const SurfN
     // compute exterior surfaces LW radiation transfer coefficient to surrounding surfaces
     // the surface.SrdSurfTemp is weighed by surrounding surfaces view factor
     Real64 HSrdSurf = 0.0;
-    auto &surface = state.dataSurface->Surface(SurfNum);
+    auto const &surface = state.dataSurface->Surface(SurfNum);
     Real64 SrdSurfsTK = surface.SrdSurfTemp + Constant::Kelvin;
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:6548]: [src/EnergyPlus/ConvectionCoefficients.cc:4691]:(style),[constVariable],Variable 'surfIntConv' can be declared as reference to const
[src/EnergyPlus/ConvectionCoefficients.cc:6547]:(style),[constVariable],Variable 'surface' can be declared as reference to const

File: src/EnergyPlus/ChillerElectricEIR.cc [hunk 24]
```diff
@@ -2403,16 +2403,16 @@ void ElectricEIRChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, b
...
-            Real64 Cp = FluidProperties::GetSpecificHeatGlycol(state,
-                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
-                                                               this->CondInletTemp,
-                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
-                                                               RoutineName);
+            Real64 CpCond = FluidProperties::GetSpecificHeatGlycol(state,
+                                                                   state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
+                                                                   this->CondInletTemp,
+                                                                   state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
+                                                                   RoutineName);
```
Comment (rraustad) [inline src/EnergyPlus/ChillerElectricEIR.cc:2410]: The other Cp used here is for the evaporator, so change this var name

[src/EnergyPlus/ChillerElectricEIR.cc:2406]:(style),[shadowVariable],Local variable 'Cp' shadows outer variable

File: src/EnergyPlus/ChillerReformulatedEIR.cc [hunk 25]
```diff
@@ -2545,16 +2545,16 @@ void ReformulatedEIRChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoa
...
-            Real64 Cp = FluidProperties::GetSpecificHeatGlycol(state,
-                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
-                                                               this->CondInletTemp,
-                                                               state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
-                                                               RoutineName);
+            Real64 CpCond = FluidProperties::GetSpecificHeatGlycol(state,
+                                                                   state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidName,
+                                                                   this->CondInletTemp,
+                                                                   state.dataPlnt->PlantLoop(this->CWPlantLoc.loopNum).FluidIndex,
+                                                                   RoutineName);
```
Comment (rraustad) [inline src/EnergyPlus/ChillerReformulatedEIR.cc:2552]: The other Cp used is for the evaporator so change the name here

[src/EnergyPlus/ChillerReformulatedEIR.cc:2548]:(style),[shadowVariable],Local variable 'Cp' shadows outer variable

File: src/EnergyPlus/BranchInputManager.cc [hunk 26]
```diff
@@ -1063,12 +1062,6 @@ namespace BranchInputManager {
                 cNumericFields.deallocate();
                 lAlphaBlanks.deallocate();
                 lNumericBlanks.deallocate();
-                if (ErrFound) {
-                    ShowSevereError(
-                        state,
-                        format("{} Invalid {} Input, preceding condition(s) will likely cause termination.", RoutineName, CurrentModuleObject));
-                    state.dataBranchInputManager->InvalidBranchDefinitions = true;
-                }
                 TestInletOutletNodes(state);
```
Comment (rraustad) [inline src/EnergyPlus/BranchInputManager.cc:1065]: [src/EnergyPlus/BranchInputManager.cc:1066]:(style),[knownConditionTrueFalse],Condition 'ErrFound' is always false

File: src/EnergyPlus/AirflowNetwork/src/Solver.cpp [hunk 27]
```diff
@@ -4607,7 +4603,7 @@ namespace AirflowNetwork {
                 int compnum = compnum_iter->second;
                 AirflowNetworkLinkageData(count).CompNum = compnum;
 
-                auto &surf = m_state.dataSurface->Surface(MultizoneSurfaceData(count).SurfNum);
+                auto const &surf = m_state.dataSurface->Surface(MultizoneSurfaceData(count).SurfNum);
```
Comment (rraustad) [inline src/EnergyPlus/AirflowNetwork/src/Solver.cpp:4607]: [src/EnergyPlus/AirflowNetwork/src/Solver.cpp:476]:(style),[knownConditionTrueFalse],Condition 'nodeErrorsFound' is always false
[src/EnergyPlus/AirflowNetwork/src/Solver.cpp:4610]:(style),[constVariable],Variable 'surf' can be declared as reference to const

File: src/EnergyPlus/AirLoopHVACDOAS.cc [hunk 28]
```diff
@@ -1008,9 +1007,7 @@ namespace AirLoopHVACDOAS {
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMaxAvail = sizingMassFlow;
             state.dataLoopNodes->Node(this->m_FanOutletNodeNum).MassFlowRateMax = sizingMassFlow;
         }
-        if (errorsFound) {
-            ShowFatalError(state, "Preceding sizing errors cause program termination");
-        }
+
         state.dataSize->CurSysNum = state.dataHVACGlobal->NumPrimaryAirSys + this->m_AirLoopDOASNum + 1;
```
Comment (rraustad) [inline src/EnergyPlus/AirLoopHVACDOAS.cc:1011]: [src/EnergyPlus/AirLoopHVACDOAS.cc:1011]:(style),[knownConditionTrueFalse],Condition 'errorsFound' is always false

**Comment by Myoldmopar** [comment]
Thanks @rraustad, I'm going to drop all of these in that are clean.

**Pull request merged**